### PR TITLE
Enable ingestion of observations by RA/Dec

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR enables ingestion of observations by RA/Dec (without needing to have field_id assigned yet).

### CODE REVIEW

File: skyportal/handlers/api/observation.py [hunk 1]
```diff
@@ -412,11 +427,31 @@ def post(self):
                 'filter',
                 'exposure_time',
             ]
+        ) and not all(
```
Comment (guynir42) [inline skyportal/handlers/api/observation.py:430]: Can we replace these with `issubset`? Also the one above it while you're at it.

File: skyportal/handlers/api/observation.py [hunk 2]
```diff
@@ -652,11 +687,31 @@ def post(self):
...
+            k in observation_data
+            for k in [
+                'observation_id',
+                'RA',
+                'Dec',
+                'obstime',
+                'filter',
+                'exposure_time',
+            ]
```
Comment (guynir42) [inline skyportal/handlers/api/observation.py:699]: issubset

**Review by guynir42 — APPROVED** [review]
Looks good. There are a few instances where we can make it more readable with `issubset` but besides that, it is good to go.

File: skyportal/handlers/api/observation.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/handlers/api/observation.py]: yessir.

File: skyportal/handlers/api/observation.py (hunk 2 above)
Comment (mcoughlin) [reply skyportal/handlers/api/observation.py]: yessir.

**Comment by mcoughlin** [comment]
Thanks @guynir42!

**Pull request merged**